rfcs/0114-Abuse protection for RabbitMQ server and tc-events.md

```diff
@@ -0,0 +1,28 @@
+# RFC 114 - Abuse protection for RabbitMQ server and tc-events.md
+* Comments: [#114](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/114)
```

Comment:
Minor process note: this will end up being RFC 118 since that's the PR number.  You can link to both #114 and #118 for comments.